addons/web/static/src/search/search_bar/search_bar.js

```diff
@@ -324,7 +324,7 @@ export class SearchBar extends Component {
      * @param {number} index
```

Comment:
Let us assume we have a slow network and a (new) search bar with two items, one of them expandable.
Before this commit, click to expand that item and then hover the other item quickly would not end in a situation
where the item is expanded. This was due to a naive concurrency management in the (new) SearchBar component.

In this commit, we allow to focus another item (via the mouse or the keyboard) without cancelling an ongoing asynchronous operation.